### PR TITLE
chore: upgrade to blobs v8

### PR DESCRIPTION
**Pull request by mrstork**
## Description

Upgrade to latest version of the blobs package and fixed up package references.
`experimentalRegion: context` is now the default option for edge deploys and it was removed from the interface.

### Documentation

Blobs v8 upgrade:
https://github.com/netlify/blobs/pull/183

Link to feature flag:
https://app.devcycle.com/o/org_dhCAK5hN1u5tXdf9/p/shared/features/6644def9bfcb11f2def7d11a

Replaces:
https://github.com/netlify/next-runtime/pull/2528

## Tests

1. `cli-before-regional-blobs-support` is failing, but it may be possible to remove this test now due to the new defaults

### CODE REVIEW

File: src/run/regional-blob-store.cts [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-export const getRegionalBlobStore = (args: Parameters<typeof getDeployStore>[0] = {}): Store => {
-  const options = typeof args === 'string' ? { name: args } : args
+export const getRegionalBlobStore = (args: GetWithMetadataOptions = {}): Store => {
   return getDeployStore({
-    ...options,
+    ...args,
     fetch: fetchBeforeNextPatchedIt,
-    experimentalRegion:
-      process.env.USE_REGIONAL_BLOBS?.toUpperCase() === 'TRUE' ? 'context' : undefined,
+    region: process.env.USE_REGIONAL_BLOBS?.toUpperCase() === 'TRUE' ? undefined : 'us-east-2',
```
Comment (mrstork) [inline src/run/regional-blob-store.cts:9]: `context` is now the default, `us-east-2` is the old default

File: src/build/plugin-context.ts [hunk 2]
```diff
@@ -153,10 +153,6 @@ export class PluginContext {
   }
 
   get useRegionalBlobs(): boolean {
-    if (!(this.featureFlags || {})['next-runtime-regional-blobs']) {
-      return false
-    }
```
Comment (mrstork) [inline src/build/plugin-context.ts:158]: flag has been fully rolled out

File: src/run/regional-blob-store.cts [hunk 3]
```diff
@@ -1,13 +1,11 @@
-import { getDeployStore, Store } from '@netlify/blobs'
+import { getDeployStore, GetWithMetadataOptions, Store } from '@netlify/blobs'
 
 const fetchBeforeNextPatchedIt = globalThis.fetch
 
-export const getRegionalBlobStore = (args: Parameters<typeof getDeployStore>[0] = {}): Store => {
-  const options = typeof args === 'string' ? { name: args } : args
+export const getRegionalBlobStore = (args: GetWithMetadataOptions = {}): Store => {
```
Comment (mrstork) [inline src/run/regional-blob-store.cts:5]: Found a nice type to use for this 👍

File: package.json [hunk 4]
```diff
@@ -49,8 +49,8 @@
   "homepage": "https://github.com/netlify/next-runtime#readme",
   "devDependencies": {
     "@fastly/http-compute-js": "1.1.4",
-    "@netlify/blobs": "^7.4.0",
-    "@netlify/build": "^29.54.3",
+    "@netlify/blobs": "^8.0.1",
+    "@netlify/build": "^29.54.2",
```
Comment (pieh) [inline package.json:53]: Was bumping min `@netlify/build` intentional?

Comment (mrstork) [reply package.json:53]: Nope! Bad merge. Good catch

**Review by pieh — APPROVED** [review]
Nice!

**Pull request merged**